kiliautoml/utils/download_assets.py

```diff
@@ -105,7 +103,7 @@ def throttled_request(api_key, asset_content):
         asset_id = asset_content.split("?AWSAccessKeyId")[0]
         return _throttled_request_memoized(api_key, asset_content, asset_id)
     else:
-        one_time_printer("Downloading public asset")  # No security token
+        one_time_logger("Downloading public asset")  # No security token
```

Comment:
Why do you need this `one_time_logger`?

Reply:
Some strings are repeated many times, the logs are overflowed by them. So I print the info just one time, and then , this only  warning is considered enough